src/pages/users/user.tsx

```
@@ -114,6 +124,27 @@ const User = () => {
   const { user } = useAuthStore();
 
+  const onFilterChange = (changedFields: FieldData[]) => {
+    console.log("changedFields", changedFields);
+
+    const changeFilterFields = changedFields
+      .map((item) => {
+        return {
+          [item.name[0]]: item.value,
+        };
+      })
+      .reduce((acc, curr) => ({ ...acc, ...curr }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)